data/cfp.md

```diff
@@ -0,0 +1,73 @@
+## DACHFest 2018 Call for Papers
+
+Thanks for your interest in speaking at DACHFest 2018! DACHFest is a community-driven conference organised by all GDGs (Google Developer Groups) in Germany, Austria and Switzerland that will take place from 10. Nov 2018 to 11. Nov 2018 at Technical University Munich in Munich, Germany. We expect about 800 attendees per day with 3 simultaneous tracks.
+
+If you have any questions, do not hesitate to contact us on Twitter or at [cfp@dachfest.com](mailto:cfp@dachfest.com).
+  
+### Deadlines
+   📅 1. September 2018 – CfP  closes 
+   📅 31. September 2018 – You will be notified about the result of your submission
+   
+### Topics 💭
+
+DACHFest is a community-driven conference open to all technologies. We are happy to hear about any topics you might think of! That includes, but is not limited to:
+  
+📱 Mobile (Android, iOS, Firebase, Machine Learning, ...)  
+💻 Web (Polymer, PWAs, Angular, Dart, new standards, tooling, ...)  
+☁️ Cloud (TensorFlow, big data, Kubernetes, security, ...)  
+📟 IoT (Android Things, Actions API, Android Wear, Chromecast, Arduino, ...)  
+🌏 Science (physics, universe, space travel, future, ...)  
+💼 Marketing & business skills targeted at indie developers or startups  
+🙋 Soft skills for developers
+
+To give you a better idea of what topics were talked about before, take a look at the sessions from the following DevFests:
+
+* [GDG DevFest Hamburg 2017](https://hamburg.devfest.de/schedule/day1)
```

Comment:
If you know more cool conferences that can give speakers a better idea of the content we aim for, feel free to add them!